plugins/nginx-vhosts/functions

```diff
@@ -128,7 +128,13 @@ validate_nginx() {
 
 restart_nginx() {
   declare desc="restart nginx for given distros"
-  fn-nginx-vhosts-nginx-init-cmd "reload"
+  local PROXY_ENABLED="$(plugn trigger proxy-is-enabled "$APP")" 
+  local PROXY_TYPE="$(plugn trigger proxy-type "$APP")"      
+  if [[ "$PROXY_ENABLED" == "true" ]]; then                  
+    if [[ "$PROXY_TYPE" == "nginx" ]]; then                
```

Comment:
Mind changing this to:

```
if [[ "$PROXY_ENABLED" == "true" ]] && [[ "$PROXY_TYPE" == "nginx" ]]; then
```

Also, there is a formatting issue, should be easy to check CI output to see whats up :)

Reply:
And done (I think)! I don't spend enough time in shell land: I wasn't confident I could stick the double.

Reply:
No worries, you'll get more comfortable as you make more pull requests :)